plugin-server/src/cdp/templates/types.ts

```diff
@@ -57,46 +58,97 @@ export const SUB_TEMPLATE_COMMON: Record<SubTemplateId, HogFunctionSubTemplate>
     'survey-response': {
         id: 'survey-response',
         name: 'Survey Response',
-        filters: {
-            events: [
-                {
-                    id: 'survey sent',
-                    type: 'events',
-                    properties: [
+        mapping_templates: [
```

Comment:
I think this is going to break things... The current Notifications tab for surveys depends on the filters field as the way to find things that are triggered by it. If we change this here it will completely break all of that...

Generally I feel this PR is doing too many things. Can you split it up into chunks instead as that way its easier to review / revert when it is individualy changes and not multiple (moving things to mappings, moving tiktok, modifiying testing etc.)

Reply:
yeah that's true. I've split this PR into three pieces:
- https://github.com/PostHog/posthog/pull/30816
- https://github.com/PostHog/posthog/pull/30819
- https://github.com/PostHog/posthog/pull/30822